manifests/service.pp

```diff
@@ -0,0 +1,31 @@
+# @summary A short summary of the purpose of this class
+#
+# A description of what this class does
+#
+# @example
+#   include pulpcore::service
+class pulpcore::service {
+  file { '/lib/systemd/system/pulpcore-api.service':
```

Comment:
We use the [systemd module](https://github.com/camptocamp/puppet-systemd) in various repositories to define these. It automatically handles all the service related parts: defining the unit file, reloading system and managing the service itself:

```puppet
systemd::unit_file { 'pulpcore-api.service':
  content => template('pulpcore/pulpcore-api.service.erb'),
  active  => true,
  enable  => true,
}
```